src/main/java/com/ita/if103java/ims/dao/impl/DashboardDaoImpl.java

```diff
@@ -27,32 +28,35 @@
 public class DashboardDaoImpl implements DashboardDao {
     private final static Logger LOGGER = LoggerFactory.getLogger(DashboardDaoImpl.class);
     private WarehouseLoadRowMapper warehouseLoadRowMapper;
-    private PopularityListRowMapper popularityListRowMapper;
+    private PopularItemsRowMapper popularItemsRowMapper;
     private EndingItemsRowMapper endingItemsRowMapper;
-    private WarehousePremiumStructRowMapper warehousePremiumLoadMapper;
+    private WarehousePremiumStructRowMapper warehousePremiumStructRowMapper;
     private WarehousePremiumLoadRowMapper warehousePremiumLoadRowMapper;
+    private ChargeCapacityRowMapper chargeCapacityRowMapper;
     private JdbcTemplate jdbcTemplate;
 
     @Autowired
     public DashboardDaoImpl(WarehouseLoadRowMapper warehouseLoadRowMapper,
                             EndingItemsRowMapper endingItemsRowMapper,
-                            PopularityListRowMapper popularityListRowMapper,
-                            WarehousePremiumStructRowMapper warehousePremiumLoadMapper,
+                            PopularItemsRowMapper popularItemsRowMapper,
+                            WarehousePremiumStructRowMapper warehousePremiumStructRowMapper,
                             WarehousePremiumLoadRowMapper warehousePremiumLoadRowMapper,
+                            ChargeCapacityRowMapper chargeCapacityRowMapper,
                             JdbcTemplate jdbcTemplate) {
-        this.warehousePremiumLoadMapper = warehousePremiumLoadMapper;
+        this.warehousePremiumStructRowMapper = warehousePremiumStructRowMapper;
         this.endingItemsRowMapper = endingItemsRowMapper;
-        this.popularityListRowMapper = popularityListRowMapper;
+        this.popularItemsRowMapper = popularItemsRowMapper;
         this.warehouseLoadRowMapper = warehouseLoadRowMapper;
         this.warehousePremiumLoadRowMapper = warehousePremiumLoadRowMapper;
+        this.chargeCapacityRowMapper = chargeCapacityRowMapper;
         this.jdbcTemplate = jdbcTemplate;
     }
     @Override
-    public List<WarehouseLoadDto> findWarehouseLoad() {
+    public List<WarehouseLoadDto> findWarehouseLoadByAccountId(Long accountId) {
         try {
-            return jdbcTemplate.query(Queries.SQL_FIND_WAREHOUSE_LOAD, warehouseLoadRowMapper);
+            return jdbcTemplate.query(Queries.SQL_FIND_WAREHOUSE_LOAD_BY_ACCOUNT_ID, warehouseLoadRowMapper, accountId);
         }catch (DataAccessException e) {
-            throw new CRUDException("DashboardDaoImpl failed to find WarehouseLoad", e);
+            throw crudException(e.toString(),"findWarehouseLoad", "accountId = " + accountId);
```

Comment:
please revert this changes back, because original exception is lost. Or pass exception object into that method